sklearn/tests/test_multiclass.py

```diff
@@ -390,7 +400,7 @@ def test_ovr_single_label_predict_proba():
     assert_almost_equal(Y_proba.sum(axis=1), 1.0)
     # predict assigns a label if the probability that the
     # sample has the label is greater than 0.5.
-    pred = np.array([l.argmax() for l in Y_proba])
+    pred = np.array([label.argmax() for label in Y_proba])
```

Comment:
Those are actually not labels, those are the raw, sample-wise predicted probabilities. But this should be further simplified to:

```suggestion
    pred = Y_proba.argmax(axis=1)
```

Reply:
Sure. I was trying to keep the existing structure. It was just the letter `l` so I thought the label is a valid replacement.